faunadb/query.py

```diff
@@ -234,6 +234,26 @@ def delete(ref):
   """See the `docs <https://faunadb.com/documentation/queries#write_functions>`__."""
   return {"delete": ref}
 
+
+def insert(ref, ts, action, params):
+  """See the `docs <https://faunadb.com/documentation/queries#write_functions>`__."""
+  return {"insert": ref, "ts": ts, "action": action, "params": params}
+
+
+def insert_event(event, params):
+  """See the `docs <https://faunadb.com/documentation/queries#write_functions>`__."""
```

Comment:
Maybe we want to document the event object taking functions to say they are just the same as the non-event object taking ones, so it's apparent to end users what they are for?